core/src/components/footer/footer.tsx

```diff
@@ -24,9 +35,46 @@ export class Footer implements ComponentInterface {
    */
   @Prop() translucent = false;
 
+  componentDidLoad() {
+    this.checkCollapsibleFooter();
+  }
+
+  componentDidUpdate() {
+    this.checkCollapsibleFooter();
+  }
+
+  private checkCollapsibleFooter = () => {
```

Comment:
Is this meant to do anything yet? Even if I set `collapse="fade"` explicitly on the `ion-footer` in the "translucent" test, it doesn't seem to have any effect, whether the footer is `translucent` or not. (I made sure to switch to `ios` mode, and I also tried fixing the issue(?) at line 90, no dice.)

Reply:
Yes, if you render `<ion-footer>` without a `collapse` prop, but then add it later this will ensure the collapsible logic gets setup properly.